Dockerfile

```diff
@@ -0,0 +1,12 @@
+# Dockerfile for instance validator, docs available at
```

Comment:
can you move this file to tools/

Reply:
The universe generation doesn't work when the file is moved to a lower directory.

Reply:
Done